t/get_next_peer.t

```diff
@@ -0,0 +1,137 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
```

Comment:
Will you please rename this file to `get-next-peer.t`? We prefer dashes over underscores in test file names. Thanks!